clone.js

```diff
@@ -1,55 +1,61 @@
-import fetch from 'node-fetch';
+// Get your free API Key at: https://www.docubee.com/solutions/integrations/docubee-api
+// Full Docubee API Documentation: https://docs.docubee.app/#overview
+
+const docubeeUrl = 'https://docubee.app/api/v2';
+
+const sourceToken = process.env.SOURCE_WORKSPACE_API_TOKEN || "SOURCE_WORKSPACE_API_TOKEN";
+const destToken = process.env.DEST_WORKSPACE_API_TOKEN || "DEST_WORKSPACE_API_TOKEN";
+
+if (sourceToken === "SOURCE_WORKSPACE_API_TOKEN" || destToken === "DEST_WORKSPACE_API_TOKEN") {
```

Comment:
This seems like an overly complicated way to do this check. We can just use the tokens as-is without the string defaults and then check for them being truthy.
```
const sourceToken = process.env.SOURCE_WORKSPACE_API_TOKEN;
const destToken = process.env.DEST_WORKSPACE_API_TOKEN

if (!sourceToken || !destToken) {
```